lib/manageiq/loggers/json_logger.rb

```diff
@@ -65,9 +65,9 @@ def translate_error(level)
         def request_id
           Thread.current[:request_id] || Thread.current[:current_request]&.request_id.tap do |request_id|
             if request_id
-              require "active_support"
-              require "active_support/deprecation"
-              ActiveSupport::Deprecation.warn("Usage of `Thread.current[:current_request]&.request_id` will be deprecated in version 0.5.0. Please switch to `Thread.current[:request_id]` to log request_id automatically.")
```

Comment:
Alternatively, I wonder if we could just remove this entirely?  Deprecated in 0.5.0 and we're v1.1.1 now

Reply:
If we want to drop the deprecated stuff when adding support for rails 7.1/7.2, we could do them in the same release.

Reply:
I think we will have to cut a major for the CloudWatch change right?  So, I'd be ok with dropping it in that case.